json_schemas/briefs-digital-outcomes-and-specialists-digital-specialists.json

```diff
@@ -188,6 +194,7 @@
     "numberOfSuppliers",
     "organisation",
     "priceWeighting",
+    "requirementsLength",
```

Comment:
I've got a PR in for the migration [here](https://github.com/alphagov/digitalmarketplace-api/pull/423). Wasn't sure about the downgrade though...